src/bio/bio_config.c

```diff
@@ -699,3 +713,96 @@ bio_set_hotplug_filter(const char *nvme_conf) {
 
 	return rc;
 }
+
+static int
+get_accel_props(const char *nvme_conf)
+{
+	struct json_config_ctx	*ctx;
+	struct spdk_json_val	*daos_data;
+	struct config_entry	 cfg = {};
```

Comment:
no init is necessary really because it gets passed to spdk_json_decode_object() which will handle the initialization but given we are initing anyway we might as well pass zero, will update